build_stubs.sh

```diff
@@ -0,0 +1,3 @@
+cd protobuf
+buf check lint && buf image build -o - | protoc --descriptor_set_in=/dev/stdin --python_out=../ $(buf image build -o - | buf ls-files --input -)
```

Comment:
What lint does? Perhaps we could use it in PR hook?

Reply:
It checks the style of the schemas, outputs a list of style violations, and returns an error code if any are found. I’d like to run it in a `pre-commit` hook, but it could also be part of CI for this repo.